partition/roles/grpc-proxy/files/grpc-proxy.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=gRPC Proxy
+After=docker.service
+Wants=docker.service
+StartLimitIntervalSec=0
+
+[Service]
+ExecStartPre=-/usr/bin/docker pull nginx:1.17
```

Comment:
Is there a specific reason why you did not use the `ansible-common/roles/systemd-docker-service` as done by the other services? This would make all systemd service files equivalent across the deployment.